support-frontend/assets/components/subscriptionCheckouts/address/addressFields.tsx

```diff
@@ -48,6 +48,13 @@ const marginBottom = css`
 	margin-bottom: ${space[6]}px;
 `;
 
+const selectStateStyles = css`
```

Comment:
Without this the select field always had a red border as soon as you rendered the page, I couldn't fathom why it was in an invalid state having not submitted the form.